### ISSUE TITLE
[enhancement]add 64bit version

### ISSUE THREAD

**Issue by chenshaoju** [cbeuw/Cloak-android#28]
Current cloak-android only has a 32-bit version, in some cases, the newest phone maybe not support it. [[The Pixel 7 series might be the first 64-bit-only Android phones](https://www.androidauthority.com/google-pixel-7-64-bit-only-3220469/)]

and some phones may show a warning message:
![QQ图片20221017111546](https://user-images.githubusercontent.com/515763/196081706-40b8de03-ac6b-4996-98ae-863db8c0e688.jpg)

Thanks for your hard work.

**Comment by fyrych** [comment]
Hi. Downloaded today, and cannot install this app. Is it possible to create a 64 bit version?

**Comment by fyrych** [comment]
> Same here. I am wondering about the effort of migrating to 64 bit. Will build with 64 bit sdk but no code change work?

I am changed ROM, installed custom rom for my phone. Problem with 32bit solved.

**Comment by bingzhangdai** [comment]
> > Same here. I am wondering about the effort of migrating to 64 bit. Will build with 64 bit sdk but no code change work?
> 
> I am changed ROM, installed custom rom for my phone. Problem with 32bit solved.

Yes, I can install the x86 version, but there are some problems when launching the shadowsocks client with cloak plugin. Every time I have to manually open the cloak plugin then return to shadowsocks client (this leaves the cloak plugin running in the background), otherwise, the shadowsocks client won't start. It will complain "unknown plugin cloak". I suspect it is due to the compatibility problems of x86 version app.

**Comment by bingzhangdai** [comment]
I can see that the build script already compiles the arm64 version apk: https://github.com/cbeuw/Cloak-android/blob/08ad33d70c368d3497c6b3226608ba9a15b6df86/app/src/make.sh#L122. I think the owner can directly put the artifact to release page?

**Comment by bingzhangdai** [comment]
These android-strip tools are removed from NDK that the cross platform compile fails. https://github.com/cbeuw/Cloak-android/blob/08ad33d70c368d3497c6b3226608ba9a15b6df86/app/src/make.sh#L96

However, "unknown plugin ck-client" is still there.

**Comment by cbeuw** [comment]
x86_64, arm, and arm64 versions of the native Go lib are included in every APK, so this shouldn't have been the issue.

I was not aware of APKs being architecture specific, I'll look into this.

**Comment by bingzhangdai** [comment]
@cbeuw This line failed https://github.com/cbeuw/Cloak-android/blob/08ad33d70c368d3497c6b3226608ba9a15b6df86/app/src/make.sh#L124 Commented these strip steps will work.
I suggest adding `set -e` in the beginning of the script to force this build step to fail in Azure DevOps. Then you can notice the error.

**Comment by cbeuw** [comment]
Ah yes you are right, it's been failing the whole time...

I'll fix the build script and make a new release

**Comment by bingzhangdai** [comment]
btw, I found the solution to "unknown plugin": https://github.com/shadowsocks/shadowsocks-android/issues/1428. It is not related to the x64 bit version.

**Comment by bingzhangdai** [comment]
@cbeuw Thank you. Another request, could you upload it to the play store? It will be more convenient, and the cellphone won't prompt the warning during installation.
